dashboard/views.py

```diff
@@ -251,7 +251,7 @@ def resource_access_within_week(request, course_id=0):
 
     # read quefrom request param
     week_num_start = int(request.GET.get('week_num_start','1'))
-    week_num_end = int(request.GET.get('week_num_end','0'))
+    week_num_end = int(request.GET.get('week_num_end','1'))
```

Comment:
I did make a change where if the UI didn't provide the dates to set the value to 1 (instead of 0) for the end date. I don't know why it was set like that and while negative days do work for this calculation it wouldn't return anything more than 0.

Reply:
Maybe these inputs should be validated someday, but I wouldn't call that high priority. The cron wouldn't pull data for dates outside of the start and end range, I don't think.